packages/open-payments/src/client/index.test.ts

```diff
@@ -97,5 +107,33 @@ describe('Client', (): void => {
         expect(error.description).toBe('Key is not a valid path or file')
       }
     })
+
+    test.each`
+      keyId          | privateKey
+      ${'my-key-id'} | ${'my-private-key'}
+      ${'my-key-id'} | ${undefined}
+      ${undefined}   | ${'my-private-key'}
+    `(
+      'throws an error if both requestInterceptor and privateKey or keyId are provided',
+      async ({ keyId, privateKey }) => {
+        try {
+          await createAuthenticatedClient({
```

Comment:
should this have a ts-ignore tag?

Reply:
Looking at it now, I expected to get a type error when running tests. However, `@swc/jest` only transpiles the code without performing any type checks.

https://swc.rs/docs/migrating-from-tsc#notes

Reply:
That's fine, it's just to not have the code lit up red in the editor :)